frontends/election-manager/src/screens/logs_screen.tsx

```diff
@@ -1,14 +1,37 @@
-import React from 'react';
-import { Prose } from '@votingworks/ui';
+import React, { useContext, useState } from 'react';
+import { Button, Prose } from '@votingworks/ui';
+import { LogFileType } from '@votingworks/utils';
 
+import { AppContext } from '../contexts/app_context';
+import { ExportLogsModal } from '../components/export_logs_modal';
 import { NavigationScreen } from '../components/navigation_screen';
 
 export function LogsScreen(): JSX.Element {
+  const { electionDefinition } = useContext(AppContext);
+  const [logFileType, setLogFileType] = useState<LogFileType>();
+
   return (
-    <NavigationScreen>
-      <Prose maxWidth={false}>
-        <h1>Logs</h1>
-      </Prose>
-    </NavigationScreen>
+    <React.Fragment>
+      <NavigationScreen>
+        <Prose maxWidth={false}>
+          <h1>Logs</h1>
+          <Button onPress={() => setLogFileType(LogFileType.Raw)}>
+            Export Log File
+          </Button>{' '}
+          <Button
+            disabled={!electionDefinition} // CDF requires the election to be known
+            onPress={() => setLogFileType(LogFileType.Cdf)}
+          >
+            Export Log File as CDF
+          </Button>
+        </Prose>
+      </NavigationScreen>
+      {logFileType && (
+        <ExportLogsModal
+          logFileType={logFileType}
+          onClose={() => setLogFileType(undefined)}
+        />
+      )}
+    </React.Fragment>
```

Comment:
Largely copied from the current admin Advanced tab: https://github.com/votingworks/vxsuite/blob/387bd5b02fa3c3444f90d8e44ba6683e6eb1f278/frontends/election-manager/src/screens/advanced_screen.tsx#L14-L49